hangman_dinner.rb

```diff
@@ -0,0 +1,201 @@
+require "colorize"
+
+class Hangman
+    def initialize
+
+      @answer_bank = ["cupcake", "spaghetti", "cookie", "hamburger", "pizza", "sandwich", "crackers", "salad","cheese", "burrito"]
+      @guesses = 4
+      @num_correct = 0
+      @current_answer = @answer_bank[rand(@answer_bank.length)]
+      @current_answer_array = @current_answer.split(//)
+      @fillable_answer = []
+      @alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K","L", "M", "N", "O", "P", "Q", "R", "S",
+      "T", "U", "V", "W", "X", "Y", "Z"]
+      @already_guessed =[]
+      @current_answer_array.length.times do |x|
+        @fillable_answer[x]= " _ "
+      end
+
+
+      draw_art
+      print_word
+      play
+
+    end
+
+    def draw_art
+
+
+      art = <<-END
+|     _________
+|     |/      |
+|     |
```

Comment:
I'm looking at you're generating the image, and I'm liking how you're using interpolation to fill in the stick figure. We can take it a step further and make it so you only have to define `art` once. The secret is you're allowed to try to print a variable that has a blank string.

So we could do this:
```
head = arms = torso = legs = ""

if @guesses == 3
  head = "(_)".colorize(:red)
elsif @guesses == 2
  head = "(_)".colorize(:red)
  arms = "\\|/".colorize(:white)  
else
...
end 

art = <<-END
          |     _________
          |     |/      |
          |     |      #{head} 
          |     |      #{arms}
          |     |       #{torso}
          |     |      #{legs}
          |_____|___________

END

puts art  
```